stellarphot/settings/custom_widgets.py

```diff
@@ -217,6 +259,13 @@ def _handle_selection(self, change):
             # but does no harm in the other cases
             self._set_disable_state_nested_models(self._item_widget, True)
 
+            # We may have arrived here by choosing a different item while
```

Comment:
Weird that you have to do this....

Reply:
It was easier than adding a cancel button...